src/terms/anyonic.jl

```diff
@@ -0,0 +1,91 @@
+# Ref https://arxiv.org/pdf/1901.10739.pdf
+# This term does not contain the kinetic energy, which must be added separately
+# /!\ They have no 1/2 factor in front of the kinetic energy,
+#     so for consistency the added kinetic energy must have a scaling of 2
+# Energy = <u, ((-i∇+βA)^2 + V) u>
+# where ∇∧A = 2π ρ, ∇⋅A = 0 => A = x⟂/|x|² ∗ ρ
+# H = (-i∇+βA)² + V - 2β x⟂/|x|² ∗ (βAρ + J)
+#   = -Δ + 2β (-i∇)⋅A + β²|A|^2 - 2β x⟂/|x|² ∗ (βAρ + J)
+# where only the first three terms "count for the energy", and where
+# J = 1/(2i) (u* ∇u - u ∇u*)
+
+struct Anyonic
+    β
```

Comment:
What is beta?

Reply:
it's not really supposed to be read without the paper

Reply:
hmm ok.